src/cert.rs

```diff
@@ -104,6 +104,16 @@ impl<'a> Cert<'a> {
                     der: CertificateDer::from(cert_der.as_slice_less_safe()),
                 };
 
+                // When used to read X509v3 Certificate.tbsCertificate.extensions, we allow
+                // the extensions to be empty.  This is in spite of RFC5280:
+                //
+                //   "If present, this field is a SEQUENCE of one or more certificate extensions."
+                //
+                // Unfortunately other implementations don't get this right, eg:
+                // - https://github.com/golang/go/issues/52319
+                // - https://github.com/openssl/openssl/issues/20877
+                const ALLOWED_EMPTY: der::AllowedToBeEmpty = der::AllowedToBeEmpty::Grudgingly;
```

Comment:
I used a const here to give a place for this comment which is not so far right. Better ideas gratefully received.

Reply:
The comment could just go on the enum definition if we're keeping it? I think `const ALLOW_EMPTY: bool = true;` with this comment would work almost as well though.